### ISSUE TITLE
Warding Spirits - (27771)

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#2349]
**In Progress Reports**

> The drop rates on quest items is wayyyy to low. Took over an hour to complete.

**Comment by Bennylavaa** [comment]
2% is far too low.

**Comment by kaytotes** [comment]
This is a direct mirror of: https://www.wowhead.com/classic/quest=8412/spirit-totem

**Thick Black Claw Drops**

- [Carrion Vulture](https://www.wowhead.com/classic/npc=1809/carrion-vulture) - 20%
- [Diseased Black Bear](https://www.wowhead.com/classic/npc=1815/diseased-black-bear) - 20%
- [Diseased Grizzly](https://www.wowhead.com/classic/npc=1816/diseased-grizzly) - 2%

Note WoWHead also has a _drastically_ lower drop rate from the grizzly's. I firmly believe this is blizzlike.

**Bloodshot Spider Eye Drops**

- [Carrion Vulture](https://www.wowhead.com/classic/npc=1809/carrion-vulture) - 20%
- [Venom Mist Lurker](https://www.wowhead.com/classic/npc=1822/venom-mist-lurker) - 20%
- [Carrion Lurker](https://www.wowhead.com/classic/npc=1821/carrion-lurker) - 2%
- [Plague Lurker](https://www.wowhead.com/classic/npc=1824/plague-lurker) - 2%

Looking at wowhead data this roughly lines up.

tl;dr working as intended, blizzard shenanigans.
